mlflow/ml-package-versions.yml

```diff
@@ -42,6 +42,8 @@ pytorch:
     requirements:
       ">= 0.0.0": ["torchvision", "scikit-learn"]
       ">= 1.8": ["transformers"]
+    python:
+      ">= 2.5.0": "3.9"
```

Comment:
Q: Will it cause an issue when we use 3.9 for all supported torch versions?

Reply:
It won't cause any issue, but should be removed once we update `requires_python` for mlflow.